### ISSUE TITLE
 TypeError: null is not an object (evaluating 'document.body.offsetHeight')

### ISSUE THREAD

**Issue by tresf** [qzind/tray#739]
@lite1979 observed while testing #731 that volvo.com's website throws the following error and prints a blank page.  Documenting here, so we can identify the cause and attempt to fix it.  Note, `document.body.offsetHeight` is code that's specific to QZ Tray.

```
	Done: 30.000000000000004 > 100.0
[TRACE] 2020-12-11 17:26:40,389 @ qz.printer.action.WebApp:?
	New state: RUNNING > SUCCEEDED
Exception in thread "JavaFX Application Thread" netscape.javascript.JSException: TypeError: null is not an object (evaluating 'document.body.offsetHeight')
	at com.sun.webkit.dom.JSObject.fwkMakeException(JSObject.java:157)
	at com.sun.webkit.WebPage.twkExecuteScript(Native Method)
	at com.sun.webkit.WebPage.executeScript(WebPage.java:1514)
	at javafx.scene.web.WebEngine.executeScript(WebEngine.java:978)
	at qz.printer.action.WebApp.findHeight(Unknown Source)
	at qz.printer.action.WebApp.lambda$static$1(Unknown Source)
	at com.sun.javafx.binding.ExpressionHelper$SingleChange.fireValueChangedEvent(ExpressionHelper.java:181)
	at com.sun.javafx.binding.ExpressionHelper.fireValueChangedEvent(ExpressionHelper.java:80)
	at javafx.beans.property.ReadOnlyObjectPropertyBase.fireValueChangedEvent(ReadOnlyObjectPropertyBase.java:74)
	at javafx.beans.property.ReadOnlyObjectWrapper.fireValueChangedEvent(ReadOnlyObjectWrapper.java:102)
	at javafx.beans.property.ObjectPropertyBase.markInvalid(ObjectPropertyBase.java:113)
	at javafx.beans.property.ObjectPropertyBase.set(ObjectPropertyBase.java:147)
	at javafx.scene.web.WebEngine$LoadWorker.updateState(WebEngine.java:1257)
	at javafx.scene.web.WebEngine$LoadWorker.dispatchLoadEvent(WebEngine.java:1372)
	at javafx.scene.web.WebEngine$PageLoadListener.dispatchLoadEvent(WebEngine.java:1237)
	at com.sun.webkit.WebPage.fireLoadEvent(WebPage.java:2514)
	at com.sun.webkit.WebPage.fwkFireLoadEvent(WebPage.java:2359)
	at com.sun.webkit.network.URLLoaderBase.twkDidFinishLoading(Native Method)
	at com.sun.webkit.network.URLLoader.notifyDidFinishLoading(URLLoader.java:775)
	at com.sun.webkit.network.URLLoader.lambda$didFinishLoading$5(URLLoader.java:766)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:428)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:427)
	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
	at java.base/java.lang.Thread.run(Thread.java:834)
[TRACE] 2020-12-11 17:26:40,400 @ qz.printer.action.WebApp:?
	Done: 100.0 > 0.0
```

**Comment by lite1979** [comment]
This time around, it printed fine. Note: there was an update to Big Sur that I applied before testing which required a restart. I blindly did it not thinking I should have tested this before the updates were installed...

**Comment by tresf** [comment]
Closed via #743.
